src/style.css

```diff
@@ -0,0 +1,104 @@
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "AvenirHeavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+.searchbar {
+  position: absolute;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  bottom: 0%;
+}
+
+.searchbar.big {
+  height: 70px;
+  top: 20px;
+}
+
+.searchbar.small {
+  height: 42px;
+  top: 110px;
+}
+
+.searchbar:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchbar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.text {
+  position: absolute;
+  border-width: 0;
+  font-family: "Avenir", sans-serif;
+  font-weight: 900;
+  color: #3d4e61;
+  width: 158px;
+}
+
+.text.big {
+  font-size: 16px;
+  line-height: 22px;
+  left: 62px;
+  top: 35.71%;
+  bottom: 37.14%;
+}
+
+.text.small {
+  font-size: 14px;
+  left: 33px;
+  top: 28.57%;
+  bottom: 26.19%;
+}
+
+.icon {
+  position: absolute;
+}
+
+.icon.big {
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
```

Comment:
after removing absolute positioning, don't forget to remove `left`, `right`, `top`, `bottom`